operator/cmd/root.go

```diff
@@ -82,6 +82,10 @@ var (
 
 		Infrastructure,
 		ControlPlane,
+
+		cell.Invoke(
```

Comment:
Let's add a comment here that this needs to be last so that everything outside leader lifecycle is started before the leader lifecycle.